ui/widgets/nuxeo-directory-radio-group.js

```diff
@@ -150,7 +150,7 @@ import { DirectoryWidgetBehavior } from './nuxeo-directory-widget-behavior.js';
     }
 
     _isChecked(entry) {
-      const value = (this.value && this.value.length > 0 ? this.value : null) || this.selectedItem;
+      const value = this.value ? this.value : this.selectedItem;
```

Comment:
So, you can remove the `this.value.length` because you initialize the variable with an empty array, right?

Reply:
That's exactly right.